python-checks/src/main/java/org/sonar/python/checks/BuiltinShadowingAssignmentCheck.java

```diff
@@ -66,6 +66,7 @@
 
   @Override
   public void initialize(Context context) {
+    variableIssuesRaised.clear();
```

Comment:
In my opinion a clear at each visit of a `FILE_INPUT` would be a bit more explicit with regard to which lifespan we want for the map's content, but I don't have that strong an opinion on this.

Reply:
Agreed, I will do that